ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionProperties.java

```diff
@@ -33,7 +42,54 @@ public IngestionReportMethod getReportMethod() {
         return reportMethod;
     }
 
-    public Map<String, String> getAdditionalProperties() {
+    public ArrayList<String> getDropByTags() {
+        return dropByTags;
+    }
+
+    public ArrayList<String> getIngestByTags() {
+        return ingestByTags;
+    }
+
+    public ArrayList<String> getAdditionalTags() {
+        return additionalTags;
+    }
+
+    public ArrayList<String> getIngestIfNotExists() {
+        return ingestIfNotExists;
+    }
+
+    public Map<String, String> getAdditionalProperties() throws IOException {
```

Comment:
Is it likely from a client application to ask for this?
If yes - it's missing javadoc
If no - access can be package-private